core/modules/cart/helpers/productChecksum.ts

```diff
@@ -1,16 +1,30 @@
 import CartItem from '@vue-storefront/core/modules/cart/types/CartItem'
 import { sha3_224 } from 'js-sha3'
 
+const replaceNumberToString = obj => {
+  Object.keys(obj).forEach(key => {
```

Comment:
You are mutating the argument here which means after calling that function it will change the argument that I passed here. I could cause problems - I've noticed your argument is eg. `bundle_options` - I don't really sure if we need any unexpected changes in there... Look at the example below, you can do something like this (no side-effects): 
```
const replaceNumberToString = obj => Object.keys(obj).reduce((prev, curr)=> ({
  ...prev,
  ...(typeof obj[curr] === 'object' ? replaceNumberToString(obj[curr]) : { [curr]: String(obj[curr]) } )
}), {})
```

Reply:
I'm just sending a clone of the object to the old function, I hope it's ok;)